pdf/Makefile

```diff
@@ -18,7 +18,11 @@ html: $(SOURCES)
 	@echo "HTML output is not yet implemented."
 
 # The version number is what follows the word "version" in the title.
-VERSION := "0.1.0"
+# PDF_VERSION should be passed as an environment variable, but will
+# default to 0.1.0 if not present.
+# An easy way to specify it at runtime would be to run:
+# PDF_VERSION=v0.3.0 make
+PDF_VERSION ?= "0.1.0"
```

Comment:
Just curious: why the change of name from "VERSION" to "PDF_VERSION" for this variable?

Reply:
Because I'm passing it as an environment variable [here](https://github.com/unicef/publicgoods-toolkit/blob/b7b8408e4edf5d6098d3e7a760414fa7fcbbabfb/.github/workflows/release.yml#L31) where I want a more descriptive name than just `VERSION` so that its purpose can be more easily identified.

Reply:
Got it, thanks.